db/seeds.rb

```diff
@@ -5,3 +5,28 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Role.find_or_create_by(name: 'observer')
+Role.find_or_create_by(name: 'client_admin')
+Role.find_or_create_by(name: 'admin')
+
+User.with_email_role_slug!(
+  ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
```

Comment:
In case I haven't mentioned, my top pet peeve is duplication; I feel more strongly that it should only be in one place than where that place is.